openc3/spec/models/target_model_spec.rb

```diff
@@ -67,12 +108,87 @@ module OpenC3
         model = TargetModel.new(folder_name: "SPEC", name: "SPEC", scope: "DEFAULT")
         model.create
         all = TargetModel.all(scope: "DEFAULT")
+        expect(all).to_not be_nil
         expect(all.keys).to contain_exactly("TEST", "SPEC")
       end
     end
 
+    describe "render" do
+      it "renders" do
+        template = '_template.erb'
+        model = TargetModel.new(folder_name: "INST", name: "INST", scope: "DEFAULT")
+        model.create
+        Dir.mktmpdir do |tmpdir|
+          tf = File.open(File.join(tmpdir, template), 'w')
+          tf.puts "CMD_LOG_CYCLE_TIME 1"
+          tf.close
+          model.render(File.expand_path(tf.path), {opt1: '1', opt2: '2', opt3: '3'})
+          # where does the rendered value go?
+        end
+      end
```

Comment:
This isn't testing anything, there are no expect statements.

Reply:
Fixed.